api/src/main/java/gov/cms/ab2d/api/controller/CapabilityStatement.java

```diff
@@ -0,0 +1,133 @@
+package gov.cms.ab2d.api.controller;
+
+import lombok.RequiredArgsConstructor;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import static gov.cms.ab2d.api.util.Constants.*;
+
+class CapabilityStatement {
+
+    private final String resourceType = "CapabilityStatement";
+
+    private final String status = "active";
+
+    private final String date = LocalDate.now().toString();
+
+    private final String publisher = "Centers for Medicare &amp; Medicaid Services";
+
+    private final String kind = "capability";
+
+    private final Software software = new Software();
+
+    private final Implementation implementation = new Implementation();
+
+    private final String fhirVersion = "4.0.1";
+
+    private final String acceptUnknown = "extensions";
+
+    private final List<String> format = List.of("application/json", "application/fhir+json");
+
+    private final List<Rest> rest = List.of(new Rest());
+
+    private final class Software {
+
+        private final String name = "AB2D";
+
+        private final String version = "latest";
+
+        private final String releaseDate = LocalDate.now().toString(); //TODO revisit this when our release schedule is figured out
+    }
+
+    private final class Implementation {
+
+        private final String url = BASE_SANDBOX_URL;
+    }
+
+    private final class Rest {
+
+        private final String mode = "server";
+
+        private final Security security = new Security();
+
+        private final List<Interaction> interaction = new ArrayList<>();
```

Comment:
Should this need be plural? `interactions`

Reply:
It's kind of weird, but it has to match up with the FHIR standard they use singular names http://hl7.org/implement/standards/fhir/capabilitystatement.html
https://bcda.cms.gov/sandbox/technical-user-guide/#examples

Reply:
Thanks for clarifying...  did not realize that earlier.

Reply:
Thanks for clarifying... did not realize that earlier.